CAIPs/caip-261.md

```diff
@@ -0,0 +1,475 @@
+---
+# Every document starts with a front matter in YAML enclosed by triple dashes.
+# See https://jekyllrb.com/docs/front-matter/ to learn more about this concept.
+caip: CAIP-261
+title: Community-powered trust assessment in software components
+author: Dayan | dayan.fc | dayan.lens | dayksx.eth | (@dayksx) <dayksx@protonmail.com>
+discussions-to: <URL(s); if multiple, list separated by , without " or []> 
+status: Draft
+type: Standard
+created: 2023-11-21
+updated: 2023-11-21
+requires (*optional): 
+---
+
+## Simple Summary
+
+CAIP-x defines a way to assess trust in discreet resources (e.g. software components or packages) by leveraging community claims and pulling in trust data from social graphs and webs of trust.
+
+## Abstract
+
+This proposal introduces a standardized data framework with the aim to propose lightweight interfaces for:
+
+1. claims made by actors in an open, peer-to-peer network about discreet resources (software components in all the examples that follow),
+2. claims made by these actors about each other,
+3. community-derived trust scores for the discreet resources and/or actors.
+
+These three data elements can be used independently of one another.
+For building a community-powered trust assessment mechanism, it's necessary to identify and gather the relevant data in accordance with the requirements of the intended trust scores computation.
+These data can then be leveraged to compute synthetic trust scores which reflect the overall sentiment of the community.
+
+The data framework modeled below incorporates the following basic primitives as inputs:
+
+- **Trust List:** This allows individuals to define their trusted peers, shaping their trust graph;
+- **Expert Report:** This enables experts to publish insights and conclusions about a given resource (e.g. an audit in the case of software components);
+- **Peer Review:** This allows individuals to endorse or dispute claims made about resources by peers;
+- **Trust Score:** This enables any trust computer to publish computed trust scores about resources and/or about the actors behind the other 3 primitives.
```

Comment:
Might be useful to define "trust computer"?

Reply:
YES PLEASE (basically the only change needed before I can approve and merge this)